packages/shared_preferences/shared_preferences/test/shared_preferences_async_test.dart

```diff
@@ -0,0 +1,912 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:shared_preferences/shared_preference_async.dart';
+import 'package:shared_preferences_platform_interface/in_memory_shared_preferences_async.dart';
+import 'package:shared_preferences_platform_interface/shared_preferences_async_platform_interface.dart';
+import 'package:shared_preferences_platform_interface/types.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
```

Comment:
I think we don't actually need this since we aren't using method channels in this package?